dotnet/src/SemanticKernel.UnitTests/Data/VolatileVectorStoreTests.cs

```diff
@@ -74,4 +87,13 @@ public sealed class SinglePropsModel<TKey>
 
         public string? NotAnnotated { get; set; }
     }
+
+    public sealed class SecondModel
```

Comment:
Make those private if they are only in the scope of this test.

```suggestion
    private sealed class SecondModel
```

Reply:
Thanks, fixed.